lib/Fmt_ast.ml

```diff
@@ -3050,15 +3053,21 @@ and fmt_case c ctx ~first ~last ~padding case =
                     fmt "@;<1 2>when " $ fmt_expression c (sub_exp ~ctx g) )
               )
           $ p.break_before_arrow $ str "->" $ p.break_after_arrow
-          $ fmt_if parens_here " (" )
+          $ fmt_if parens_here
+              ( match grouping with
+              | `Parens -> " ("
+              | `Begin_end -> "@;<1 0>begin" ) )
```

Comment:
Can't this be moved into `Params.get_cases` ?
Also, that's yet an other piece of code rendering begin/end, I'd prefer if we can find a way to handle that in a consistent way (from `Params` maybe ?).

Reply:
done!